Compiling using Maven 4.0.0-rc5 fails, due to incorrect pom.xml file

**Describe the bug**

I'm trying to build using apache-maven-4.0.0-rc-5, just to see if there were any issues compared to Maven 3.
The build stops with the error message

```
11:39:00,049 [ERROR] Failed to execute goal io.quarkus:quarkus-extension-maven-plugin:3.30.2:extension-descriptor (default) on project operaton-bpm-quarkus-engine: Execution default of goal io.quarkus:quarkus-extension-maven-plugin:3.30.2:extension-descriptor failed: The following errors were encountered while processing Quarkus application dependencies:
11:39:00,049 [ERROR] 1) Failed to load POM from /home/alf/devel/github/operaton/qa/tomcat-runtime/pom.xml
11:39:00,049 [ERROR]   at io.quarkus.bootstrap.resolver.maven.workspace.WorkspaceLoader.readModel(WorkspaceLoader.java:63)
11:39:00,049 [ERROR]   ...
11:39:00,049 [ERROR] -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal io.quarkus:quarkus-extension-maven-plugin:3.30.2:extension-descriptor (default) on project operaton-bpm-quarkus-engine: Execution default of goal io.quarkus:quarkus-extension-maven-plugin:3.30.2:extension-descriptor failed: The following errors were encountered while processing Quarkus application dependencies:
1) Failed to load POM from /home/alf/devel/github/operaton/qa/tomcat-runtime/pom.xml
```

**To Reproduce**

Steps to reproduce the behavior:
Put Maven 4 on your PATH
In the top directory of Operaton, do "mvn clean install -DskipTests=true"
Wait while the Maven is building, in the end it will stop with error message

**Expected behavior**
The Maven build completes with a success.

**Additional context**
Running "maven -X clean install -DskipTests=true" gave more information, like this
11:43:26,406 [WARNING] [stderr] Caused by: org.codehaus.plexus.util.xml.pull.XmlPullParserException: ParseError at [row,col]:[286,13]
11:43:26,406 [WARNING] [stderr] Message: Unrecognised tag: '{http://maven.apache.org/POM/4.0.0}goals' caused by: javax.xml.stream.XMLStreamException: ParseError at [row,col]:[286,13]
11:43:26,406 [WARNING] [stderr] Message: Unrecognised tag: '{http://maven.apache.org/POM/4.0.0}goals'
11:43:26,406 [WARNING] [stderr] 	at org.apache.maven.model.io.xpp3.MavenXpp3Reader.read(MavenXpp3Reader.java:113)
11:43:26,406 [WARNING] [stderr] 	at org.apache.maven.model.io.xpp3.MavenXpp3Reader.read(MavenXpp3Reader.java:128)
11:43:26,406 [WARNING] [stderr] 	at org.apache.maven.model.io.xpp3.MavenXpp3Reader.read(MavenXpp3Reader.java:141)
11:43:26,406 [WARNING] [stderr] 	at io.quarkus.bootstrap.resolver.maven.workspace.ModelUtils.readModel(ModelUtils.java:242)
11:43:26,406 [WARNING] [stderr] 	... 12 more
11:43:26,406 [WARNING] [stderr] Caused by: javax.xml.stream.XMLStreamException: ParseError at [row,col]:[286,13]



So the reason is that the qa/tomcat-runtime/pom.xml has this content
          <plugin>
            <groupId>org.apache.maven.plugins</groupId>
            <artifactId>maven-antrun-plugin</artifactId>
            <goals>
              <goal>run</goal>
            </goals>
            <executions>
              <execution>
                <id>clean-webapps</id>
                <phase>process-resources</phase>
                <goals>
                  <goal>run</goal>
                </goals>
                <configuration>


The first "goals" section is misplaced, the goals should only be inside the execution element.
Maven 4 is more strict on parsing the pom.xml files.

--- a/qa/tomcat-runtime/pom.xml
+++ b/qa/tomcat-runtime/pom.xml
@@ -283,9 +283,6 @@
           <plugin>
             <groupId>org.apache.maven.plugins</groupId>
             <artifactId>maven-antrun-plugin</artifactId>
-            <goals>
-              <goal>run</goal>
-            </goals>
             <executions>
               <execution>
                 <id>clean-webapps</id>

